uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/FloorFilterController.qml

```diff
@@ -0,0 +1,703 @@
+        } else if (automaticSelectionMode === FloorFilterController.AutomaticSelectionMode.Always) {
+            setSelectedFacilityId("");
+            setSelectedLevelId("");
+        }
+    }
+
+    /*!
+      \qmlproperty GeoModel geoModel
+      \brief The geomodel the controller is listening for \c FloorManager changes.
+    */
+    property GeoModel geoModel: {
+        // check geoView has been set
+        if (!geoView)
+            return null;
+        if (typeof (geoView.map) !== "undefined")
+            return geoView.map;
+        else if (typeof (geoView.scene) !== "undefined")
+            return geoView.scene;
+        return null;
+    }
+
+    onGeoModelChanged: {
+        // make sure that the geomodel is loaded.
+        geoModel.load();
+    }
+
+    /*!
+      \qmlproperty int updateLevelsMode
+      \brief The mode that defines how levels are made visible/invisible in the geoView.
+      \value UpdateLevelsMode.AllLevelsMatchingVerticalOrder When a level is selected, all levels with matching vertical order are made visible, all other levels are invisible.
+      \value UpdateLevelsMode.SingleLevel All levels with \c{verticalOrder == 0} are set to visible, the currently selected level is also visible, all other levels are set to invisible. This
+      matches the JavaScript Widget FloorFilter functionality.
+
+      Default is \c{UpdateLevelsMode.AllLevelsMatchingVerticalOrder}
+    */
+    property int updateLevelsMode: FloorFilterController.UpdateLevelsMode.AllLevelsMatchingVerticalOrder
+
+    /*!
+      \qmlproperty FloorManager floorManager
+      \brief The FloorManager associated to the \l geoModel.
+      It can be null if no associated geoModel is available or is not loaded.
+    */
+    property FloorManager floorManager : geoModel? geoModel.floorManager : null
+
+    onFloorManagerChanged: {
+        if (floorManager.loadStatus !== Enums.LoadStatusLoaded)
+            floorManager.load();
+        else{
+            if(floorManager.sites.length > 0)
+                populateSites(floorManager.sites);
+            else {
+                populateAllFacilities();
+            }
+
+            controller.loaded(); //emit the loaded signal
+        }
+    }
+
+    /*!
+      \qmlproperty bool selectedSiteRespected
+      \brief Repopulates the \l facilities model when toggled false.
+      When the selected site is respected, the facilities list is limited to the facilities which match the selected site.
+      If the selected site is not respected, then facilities will contain all facilities from all sites, regardless of the selected site.
+      Default is {true}.
+    */
+    property bool selectedSiteRespected: true
+
+    onSelectedSiteRespectedChanged: {
+        if (!selectedSiteRespected) {
+            populateAllFacilities();
+        }
+    }
+    
+    /*!
+      \qmlproperty string selectedLevelId
+      \brief Stores the current selected level Id.
+      Default is {""}.
```

Comment:
Missing `\c`?

Reply:
You mean like this:
`\brief Stores the current \c string selected level Id.`

Reply:
Sorry I meant:

```
Default is \c{""}.
```